app/controllers/api/v1/podcasts_controller.rb

```diff
@@ -15,7 +15,14 @@ def create
     end
 
     def show
-        render json: Podcast.find(params[:id])
+        podcast = Podcast.find(params[:id])
+        render json: {
+            podcast: serialized_data(podcast, PodcastSerializer)
+        }
+    end
+
+    def serialized_data(data, serializer)
+        ActiveModelSerializers::SerializableResource.new(data, each_serializer: serializer)
     end
```

Comment:
love that you are adding a serializer here, but you should only need line ` render json: Podcast.find(params[:id])`, you shouldned need this...interesting method

Reply:
your serializer is looking good, so ping me if you want to sync